Results: add initial set of results (#16)
greenelab/manubot-gpt-manuscript: content/04.results.md

> # Evaluating the AI-based revision workflow {#sec:results}

Reviewer: ```suggestion
# Observations of AI-based revisions {#sec:results}
```

Reviewer: I'm not sure this is quite the right thing - essentially you're describing case studies - maybe we should use that language? I like where this is going - it's a bit of observations from the earliest days of this type of tool... but maybe we need a different term like case studies & acknowledge that the review is subjective.

Author: Perfect! I agree.

Then, we ran the workflow on the three manuscripts using the three language models described above, producing one pull request (PR) per manuscript and model.
These PRs (three per manuscript) can be accessed from the "Pull requests" tab from each repository, where they are titled *"GPT (MODEL) used to revise manuscript"* with *MODEL* being the identifier of the model used.
PRs show the differences between the original text and the suggestions made by the AI-based revision tool.
Below we discussed our findings based on these PRs using the language models across different sections of the manuscripts.

## Performance of language models

We found that Davinci models, as expected, were superior than the Curie model for all manuscripts.
The Curie model is described as "very capable", and it is faster and less expensive than Davinci models.
However, as shown in the PRs generated using this model (titled `GPT (text-curie-001) used to revise manuscript`), the model was not able to produce acceptable revisions for any of the manuscripts.
Most of its suggestions were not coherent with the original text in any of the sections.

> Among Davinci models, we found that for `text-davinci-edit-001` (edits endpoint), the quality of the revisions was far inferior than `text-davinci-003` (completion endpoint).

Reviewer: ```suggestion
Among Davinci models, we found that for `text-davinci-edit-001` (edits endpoint), the quality of the revisions was subjectively inferior to `text-davinci-003` (completion endpoint).
```
Softened given the limited evaluation.
